tests/subscriptions/test_codecs.py

```diff
@@ -61,45 +61,38 @@ def test_decode(self):
         assert codec.decode(payload) == subscription
 
 
-class TestSubcriptionTaskResultCodec(BaseTest):
-    def test_encode(self):
-        timestamp = datetime.now()
+def test_subscription_task_result_encoder() -> None:
+    timestamp = datetime.now()
 
-        subscription_data = SubscriptionData(
-            1,
-            [],
-            [["count()", "", "count"]],
-            timedelta(minutes=1),
-            timedelta(minutes=1),
-        )
+    subscription_data = SubscriptionData(
```

Comment:
did this format differently before just because of the additional indentation ? (it ishard to judge this on github)

Reply:
I think so. This view helps: https://github.com/getsentry/snuba/pull/1126/files/b85b010030e1a863a2afd69307d262a54c3189c7?diff=split&w=1